### PR TITLE
EPMRPP-37186 React. Adjust Email notification component

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/controllers/project/actionCreators.js [hunk 1]
```diff
@@ -40,25 +40,16 @@ export const updateProjectFilterPreferencesAction = (filterId, method) => (dispa
...
-  const integrationName = notificationIntegrationNameSelector(getState());
-  const newConfig = {
-    integrationName,
-    enabled,
-    integrationParameters: {
-      rules,
-    },
-  };
-  fetch(URLS.projectIntegration(activeProjectSelector(getState())), {
+export const updateProjectNotificationsConfig = (NotificationsConfig) => (dispatch, getState) => {
```
Comment (DiscoElevator) [inline app/src/controllers/project/actionCreators.js]: `notificationsConfig`

File: app/src/controllers/project/selectors.js [hunk 2]
```diff
@@ -65,21 +62,24 @@ export const notificationIntegrationEnabledSelector = (state) =>
...
 
-export const notificationRulesSelector = createSelector(
-  notificationIntegrationParametersSelector,
-  ({ rules = [] }) =>
-    rules.map((rule) => ({
-      ...rule,
-      informOwner: rule.recipients.includes(OWNER),
+export const projectNotificationsCasesSelector = createSelector(
+  projectNotificationsConfigurationSelector,
+  ({ cases }) =>
```
Comment (DiscoElevator) [inline app/src/controllers/project/selectors.js]: `{ cases = [] }`

File: app/src/controllers/project/actionCreators.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/controllers/project/actionCreators.js]: Fixed

File: app/src/controllers/project/selectors.js (hunk 2 above)
Comment (AmsterGet) [reply app/src/controllers/project/selectors.js]: Fixed

**Pull request merged**